### PR TITLE
(docs) Pull in updates to auto-generated docs for plan functions

### PR DESCRIPTION
**Pull request by donoghuc**
Update to yard gem appears to have changed the way puppet-strings generates the docs. This commit updates the plan reference based on those changes.

### CODE REVIEW

**Comment by donoghuc** [comment]
Put this up just to see how it renders in markdown. Pinning `yard` to `0.9.19` seems to change the output. Still evaluating if we want to pull in these changes.

File: pre-docs/plan_functions.md [hunk 1]
```diff
@@ -1,6 +1,177 @@
 # Bolt Functions
 
 
+## :'ctrl::do_until'
```
Comment (donoghuc) [inline pre-docs/plan_functions.md]: Oh, boo it prepends non-core functions with a `:` which I think changes the order. Perhaps we should pin yard?

Comment (lucywyman) [reply pre-docs/plan_functions.md]: I'm fine with pinning yard if that fixes it. I also don't think it's a huge issue if it's not quick to fix. I'm more urgent about getting travis passing.

File: Rakefile [hunk 2]
```diff
@@ -88,7 +88,8 @@ task :docs do
   #       text: example body
   @functions = funcs.map do |func|
     func['text'] = func['docstring']['text']
-
+    # TODO: remove once puppet-strings releases fix for :'file::read'
+    func['name'] = func['name'][1..-1].gsub("'", '') if func['name'].start_with?(':')
```
Comment (donoghuc) [inline Rakefile]: If this had not affected ordering I would be more inclined to do it. so i think pinning makes sense. https://github.com/puppetlabs/bolt/pull/1062

**Comment by donoghuc** [comment]
Chose to pin yard for now. Will follow up on a fix with puppet-strings.

**Pull request closed without merging**